modules/schema/src/main/java/org/apache/ignite/internal/schema/row/Row.java

```diff
@@ -894,8 +897,21 @@ private byte[] readBytes(ByteBuffer chunk, int off, int len) {
         }
     }
 
+    /** {@inheritDoc} */
     @Override
     public byte[] bytes() {
         return row.bytes();
     }
+
+    /** {@inheritDoc} */
+    @Override
+    public int colocationHash() {
+        HashCalculator hashCalc = new HashCalculator();
```

Comment:
do we really need to calculate it every time?

Reply:
Fixed